pkg/logs/client/http/destination.go

```diff
@@ -232,7 +237,13 @@ func buildURL(endpoint config.Endpoint) string {
 	} else {
 		address = endpoint.Host
 	}
-	return fmt.Sprintf("%v://%v/v1/input", scheme, address)
+	uri := ""
+	if endpoint.Version == config.EPIntakeVersion2 && endpoint.TrackType != "" {
+		uri = fmt.Sprintf("api/v2/%s", endpoint.TrackType)
+	} else {
+		uri = "v1/input"
+	}
+	return fmt.Sprintf("%v://%v/%v", scheme, address, uri)
```

Comment:
Nits,
I think you can use the [net/url](https://golang.org/pkg/net/url/#URL) package here to build the URI.

Reply:
Good catch, thanks!